pandas/core/indexes/multi.py

```diff
@@ -1182,6 +1182,8 @@ def to_frame(self, index=True):
 
     def to_hierarchical(self, n_repeat, n_shuffle=1):
         """
```

Comment:
to_hierarchical is listed elsewhere in this file in a doc-string

Reply:
Thanks - it is mentioned as a part of enumeration of methods of `MultiIndex` - should it be removed?

Reply:
Thanks - updated for all of the latest review comments